exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -97,32 +113,43 @@ class JaegerSpanExporter(SpanExporter):
             when query for spans.
         agent_host_name: The host name of the Jaeger-Agent.
         agent_port: The port of the Jaeger-Agent.
-        collector_endpoint: The endpoint of the Jaeger-Collector HTTP/HTTPS Thrift.
+        collector_endpoint: The endpoint of the Jaeger-Collector
```

Comment:
Is `collector_endpoint` the endopoint of the Jaeger collector that uses Thrift over HTTP/HTTPS or Protobuf via gRPC? If so, I would suggest these changes.
```suggestion
        collector_endpoint: The endpoint of the Jaeger collector that uses
```